cmd/system-probe/app/restart.go

```diff
@@ -0,0 +1,51 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016-present Datadog, Inc.
+
+package app
+
+import (
+	"fmt"
+	"io/ioutil"
+
+	"github.com/DataDog/datadog-agent/cmd/system-probe/api"
+	"github.com/spf13/cobra"
+)
+
+func init() {
+	SysprobeCmd.AddCommand(moduleRestartCommand)
+}
+
+var moduleRestartCommand = &cobra.Command{
+	Use:   "module-restart [module]",
+	Short: "Restart a given system-probe module",
+	Long:  ``,
```

Comment:
I think we can do that by importing the modules package and using the `modules.All` variable. will give it a try!

Reply:
Unfortunately I don't it's possible to do this dynamically because `moduleRestartCommand` gets evaluated before I can retrieve the module names. I guess I could try to declare the command in a different way (like setting the `Short` field in the `init()` function) but I personally lean towards keeping this consistent with the other command declarations we have already.
So if you're fine with it I'll leave it as it is.

Reply:
That's fine by me!